src/components/ContentSidebar/ActivityFeed/task-form/index.js

```diff
@@ -0,0 +1,3 @@
+// @flow
```

Comment:
We're still following PascalCase for directory names, at least for the time being.

Reply:
not in this directory, this matches what's there